test/test_encryption.py

```diff
@@ -2099,5 +2099,92 @@ def test_05_roundtrip_encrypted_unindexed(self):
         self.assertEqual(decrypted, val)
 
 
+class TestQueryableEncryptionDocsExample(EncryptionIntegrationTest):
+    # Queryable Encryption is not supported on Standalone topology.
+    @client_context.require_no_standalone
+    @client_context.require_version_min(6, 0, -1)
+    def setUp(self):
+        super().setUp()
+
+    def test_queryable_encryption(self):
+        # Drop data from prior test runs.
+        MongoClient().keyvault.datakeys.drop()
+        MongoClient().docs_examples.encrypted.drop()
+
+        kms_providers_map = {"local": {"key": LOCAL_MASTER_KEY}}
+
+        # Create two data keys.
+        key_vault_client = MongoClient()
```

Comment:
All the MongoClient()s in this test need to be replaced with rs_or_single_client so that they work with auth/tls/etc... We use this pattern in other docs example tests:
```
        def MongoClient(**kwargs):
            c = rs_or_single_client(**kwargs) 
            self.addCleanup(c.close)
            return c
```

Reply:
Done